app/models/katello/sync_plan.rb

```diff
@@ -118,7 +118,7 @@ def sync_time
     end
 
     def next_sync_date
-      return nil unless (self.enabled || !self.foreman_tasks_recurring_logic.tasks.nil?)
+      return nil unless (self.enabled || !self.foreman_tasks_recurring_logic.try(:tasks).nil?)
```

Comment:
can you flip this to `self.foreman_tasks_recurring_logic.try(:tasks).present?`? this would avoid a double negative